ci/environment.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
   - toml
   - pytest
   - jupyter
-  - xeus=1.0.1
+  - xeus
```

Comment:
Is it necessary to pin?

Reply:
Yes, I believe... The modules or functions used by LFortran might be taken from the xeus 1.0.1 version.